docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -8,9 +8,14 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 include::./attributes.adoc[]
 
 
-Quarkus includes the `kubernetes-client` extension which enables the use of the [Fabric8 Kubernetes Client](https://github.com/fabric8io/kubernetes-client)
+Quarkus includes the `kubernetes-client` extension which enables the use of the link:https://github.com/fabric8io/kubernetes-client[Fabric8 Kubernetes Client]
```

Comment:
No need to fix it but for the future, you only need link if it's not obvious it's a URL (so things starting with `https://` or `http://` don't need it).

Reply:
Good to know!